src/games/Lexible/models/ClientModel.ts

```diff
@@ -190,38 +214,21 @@ export class LexibleClientModel extends ClusterfunClientModel  {
         }
     }
 
-    // -------------------------------------------------------------------
-    // handleScoredWordMessage
-    // -------------------------------------------------------------------
-    protected handleScoredWordMessage = (message: LexibleScoredWordMessage) => {
+    protected handleBoardUpdateMessage = (sender: string, message: LexibleBoardUpdateNotification) => {
         message.letters.forEach(l => {
             const block = this.theGrid.getBlock(l.coordinates)
             if(!block) Logger.warn(`WEIRD: No block at ${l.coordinates}`)
-            else block.setScore( Math.max(message.score, block.score), message.team);
+            else block.setScore( Math.max(message.score, block.score), message.scoringTeam);
         })
         this.updateWinningPaths();
         this.saveCheckpoint();
-        this.ackMessage(message);  
-        this.invokeEvent(LexibleGameEvent.WordAccepted)
-    }
-
-    // -------------------------------------------------------------------
-    // handleFailedWordMessage
-    // -------------------------------------------------------------------
-    protected handleFailedWordMessage = (message: LexibleFailedWordMessage) => {
-        message.letters.forEach(w => {
-            const block = this.theGrid.getBlock(w.coordinates)
-            if(!block) {
-                Logger.warn(`WEIRD: No block at ${JSON.stringify(w.coordinates)}`)
-            }
-            else block.fail()
-        })
+        this.invokeEvent(LexibleGameEvent.WordAccepted);
     }
 
     // -------------------------------------------------------------------
     // handleRecentlyTouchedMessage
     // -------------------------------------------------------------------
-    protected handleRecentlyTouchedMessage = (message: LexibleRecentlyTouchedLettersMessage) => {
+    protected handleRecentlyTouchedMessage = (sender: string, message: LexibleRecentlyTouchedLettersMessage) => {
```

Comment:
I decided the ergonomics were better this way, in having the receiver function take an extra sender parameter instead of forcing the sent message to conform to `{ sender: string }` everywhere. Plus, it means that messages can be anything, including bare strings or numbers.